packages/web3-eth/src/rpc_method_wrappers.ts

```diff
@@ -1737,49 +980,8 @@ export async function estimateGas<ReturnFormat extends DataFormat>(
 
 // TODO - Add input formatting to filter
 /**
- * Gets past logs, matching the provided `filter`.
- *
+ * View additional documentations here: {@link Web3Eth.getLogs}
```

Comment:
```suggestion
 * View additional documentations here: {@link Web3Eth.getPastLogs}
```
as`@ link Web3Eth.getLogs` navigation doesn't work